### ISSUE TITLE
Blueprint - Custom Karaf Commands

### ISSUE THREAD

**Issue by bsaunder** [rhtconsulting/fuse-quickstarts#39]
Example of writting a Custom Karaf Command

**Comment by paoloantinori** [comment]
does this cover it? https://github.com/paoloantinori/sample-karaf-command-with-blueprint

**Comment by mmistretta** [comment]
Yes that will cover it.  Any chance you could submit a Pull Request to get it incorporated into the larger project?  Thanks!

**Comment by sabre1041** [comment]
@paoloantinori I have an example project that I can pass along on this topic if needed
